test/__mocks__/session.mock.ts

```diff
@@ -864,6 +868,21 @@ export const TRUST_PARTIAL_DATE: Trust = {
   unable_to_obtain_all_trust_info: "No"
 };
 
+export const OVER_SEAS_ENTITY_MOCK_DATA: ICompanyDetails = {
```

Comment:
This would need to change to CompanyProfile !

Reply:
just remove for now